api/apod.py

```diff
@@ -0,0 +1,49 @@
+import os
+
+import requests
+from flask import jsonify, Blueprint, current_app
+
+from api.request import Request
+from api.schemas import PayloadSchema
+from api.utils import (
+    save_image,
+    get_jwt,
+    get_json,
+    get_media_path,
+    get_file_name
+)
+
+apod_api = Blueprint('apod', __name__)
+
+
+@apod_api.route('/apod', methods=['POST'])
+def apod():
+    url = current_app.config['APOD_URL']
+    params = {'api_key': get_jwt(),
+              **get_json(PayloadSchema())}
+
+    apod_request = Request(url, params=params)
+
+    response = apod_request.get().json()
+
+    if 'thumbnail_url' in response:
+        image_url = response['thumbnail_url']
+    else:
+        image_url = (
+            response['hdurl'] if bool(params['hd']) is True
```

Comment:
Please, don't compare boolean values to True or False using == or is operator. Read and use https://www.python.org/dev/peps/pep-0008/ it’s really easier for other developers to read your code when you do so.